spec/workers/slack_id_import_worker_spec.rb

```diff
@@ -10,8 +10,12 @@
   context 'occurs weekly' do
     it 'occurs at expected time' do
       scheduled_job
-      assert_equal true, described_class.jobs.last['jid'].include?(scheduled_job)
-      expect(described_class).to have_enqueued_sidekiq_job('Awesome', true).at(time)
+      assert_equal true, described_class
```

Comment:
nit:
I prefer to not mix minitest assertions and rspec matchers.
Being consistent makes code more readable.